cfgov/cfgov/wsgi.py

```diff
@@ -10,6 +10,24 @@
 import os
 
 
+import dotenv
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def initialize_environment():
+    environment_candidates = [
```

Comment:
Why do we need these 3 possible options?  Seems like a chance to have a `.env` in more than one place, and mistake which one is actually being used.  Seems worth of a comment and/or some docs describing when to use these different locations.

Reply:
Agree, it'd be nice to standardize on one location for .env. Can we standardize on using the repo root (`../.env`), which would also work out to the DFD root during deployments?

Reply:
That makes sense to me.

Reply:
it now only looks for '.env' as a sibling of 'cfgov'